packages/ui-stencil/src/components/internal/orama-logo-message-boxes-icon/orama-logo-message-boxes-icon.tsx

```diff
@@ -0,0 +1,224 @@
+import { Host, Component, Prop, h } from '@stencil/core'
```

Comment:
@albertogiunta you may have considered the orama-logo-icon as a reference for this component, but actually we have to refactor it in order to use the icons functional component (see `icons.tsx`). This component is stateless so we can simplify and just use the functional approach